Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -338,6 +339,13 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
                         "dimension");
             }
         }
+        else if (type == "DensityDrivenFlow")
```

Comment:
Hi all, I just discussed with @TomFischer and think, that we should call its something like HT, HC, or HTC. ´´´DensityDrivenFlow´´´ is not complete in its description, as we may have additional variables to change, like viscosity. Also, the processes may simply be used with constant variables for "simple" solute/heat transport.